perftest/spreadtestnd.cpp

```diff
@@ -1,301 +1,265 @@
-#include "finufft/finufft_utils.hpp"
-#include <finufft/spreadinterp.h>
-#include <finufft/test_defs.h>
-
-#include <math.h>
-#include <stdio.h>
-#include <stdlib.h>
+#include <cmath>
+#include <cstdio>
+#include <numeric>
 #include <vector>
 
-#include "finufft/finufft_utils.hpp"
-using namespace finufft::spreadinterp;
+#include <finufft/test_defs.h>
+
 using namespace std;
 using namespace finufft::utils;
 
-void usage() {
-  printf("usage: spreadtestnd dims [M N [tol [sort [flags [debug [kerpad [kerevalmeth "
-         "[upsampfac]]]]]]]]\n\twhere dims=1,2 or 3\n\tM=# nonuniform pts\n\tN=# uniform "
-         "pts\n\ttol=requested accuracy\n\tsort=0 (don't sort NU pts), 1 (do), or 2 "
-         "(maybe sort; default)\n\tflags: expert timing flags, 0 is default (see "
-         "spreadinterp.h)\n\tdebug=0 (less text out), 1 (more), 2 (lots)\n\tkerpad=0 (no "
-         "pad to mult of 4), 1 (do, for kerevalmeth=0 only)\n\tkerevalmeth=0 (direct), 1 "
-         "(Horner ppval)\n\tupsampfac>1; 2 or 1.25 for Horner\n\nexample: ./spreadtestnd "
-         "1 1e6 1e6 1e-6 2 0 1\n");
+/* clang-format off */
+static void usage() {
+  printf(
+    "usage: spreadtestnd dims [M N [tol [sort [flags [debug [kerpad [kerevalmeth [upsampfac]]]]]]]]\n"
+    "\twhere dims=1,2 or 3\n"
+    "\tM=# nonuniform pts\n"
+    "\tN=# uniform pts (rough total; per-dim N=round(N^(1/d)))\n"
+    "\ttol=requested accuracy\n"
+    "\tsort=0 (no), 1 (yes), 2 (auto; default)\n"
+    "\tflags: expert timing flags, 0 default (see spreadinterp.h)\n"
+    "\tdebug=0,1,2\n"
+    "\tkerpad=0 (no pad), 1 (pad; for kerevalmeth=0 only)\n"
+    "\tkerevalmeth=0 (direct), 1 (Horner)\n"
+    "\tupsampfac>1; 2 or 1.25 typical for Horner\n"
+    "\nexample: ./spreadtestnd 3 8e6 8e6 1e-6 2 0 1\n");
```

Comment:
Thanks! Yeah, that was a one of the things that made me upset about enforcing clang format months ago...  I assume that can't happen any more?